node/cli/Cargo.toml

```diff
@@ -94,11 +95,12 @@ futures03 = { package = "futures-preview", version = "0.3.0-alpha.19" }
 tempfile = "3.1.0"
 
 [build-dependencies]
-substrate-cli = { git = "https://github.com/darwinia-network/substrate.git", branch = "darwinia-develop", optional = true }
 build-script-utils = { package = "substrate-build-script-utils", git = "https://github.com/darwinia-network/substrate.git", branch = "darwinia-develop" }
 structopt = "0.3.3"
 vergen = "3.0.4"
 
+darwinia-cli = { path = "../../core/cli", optional = true }
```

Comment:
Is this duplicated?

Reply:
The origin file contains this line, I just replace all `substrate-cli` with `darwinia-cli`.